packages/core/src/testE2E/codecatalyst/client.test.ts

```diff
@@ -425,15 +429,7 @@ describe('Test how this codebase uses the CodeCatalyst API', function () {
      */
     async function createTestCodeCatalystClient(auth: Auth): Promise<CodeCatalystClient> {
         const conn = await useCodeCatalystSsoConnection(auth)
-        return await createCodeCatalystClient(conn, undefined, undefined, {
-            // Add retries for tests since many may be running in parallel in github CI.
-            // AWS SDK adds jitter automatically.
-            // https://github.com/aws/aws-sdk-js/blob/3e616251947c73d5239178c167a9d73d985ca581/lib/util.js#L884
-            retryDelayOptions: {
-                base: 1200, // ms
-            },
-            maxRetries: 5,
-        })
+        return await createCodeCatalystClient(conn, undefined, undefined)
```

Comment:
> Remove retry logic in tests, since V3 client builder adds this by default.

what are the default values, do they match the removed values here? May be useful to be explicit here in any case.

Reply:
Before it made up to 5 attempts with a constant delay. The new default is 5 attempts with exponential backoff implemented here: https://github.com/aws/aws-toolkit-vscode/blob/4c190f4d9678a3bae38ab56000f5ff6123eea6e2/packages/core/src/shared/awsClientBuilderV3.ts#L150